mm2src/coins/nft.rs

```diff
@@ -103,7 +103,7 @@ pub async fn get_nft_metadata(_ctx: MmArc, req: NftMetadataReq) -> MmResult<Nft,
     let token_address = &req.token_address;
     let token_id = &req.token_id;
     let uri = format!(
-        "{req_url}{MORALIS_API_ENDPOINT}nft/{token_address}/{token_id}?chain={chain_str}&{FORMAT_DECIMAL_MORALIS}"
+        "{req_url}{MORALIS_API_ENDPOINT}nft/{token_address:#02x}/{token_id}?chain={chain_str}&{FORMAT_DECIMAL_MORALIS}"
```

Comment:
Since `req_url` is a `Url` type, we can use `join` method instead of building string here.

Reply:
Done